src/README.md

```diff
@@ -0,0 +1,4 @@
+Requirements to build
+- Visual Studio 2022 17.6.3 or above
+- .Net SDK 6.0.410
```

Comment:
>.Net SDK 6.0.410 [](http://example.com/codeflow?start=2&length=16)

New requirement

Reply:
IF this is upgrading tests to same as rest of project; why is there a new build requirement?